Minutes/2024-01-15.md

```diff
@@ -0,0 +1,113 @@
+
+## Attendance
+
+* Record attendees: James, Phebe, Karen, Mahmoud, Moshe
+* Record absences: None
+* Vote to approve 2023-11-06 minutes:
+  * Mahmoud motions
+  * Karen seconds
+  * For: James, Phebe, Karen, Mahmoud, Moshe
+  * Motions passes
+* Ask for any additions to the agenda: None
+
+## PyBay 2024 timeline
+
+* Phebe: Second half of September for the event.
+  * Ticket sales until date-of or two weeks
+  * Month before to approve speakers: End of July
+  * Two weeks to a month to choose speakers: Start of July
+  * Karen: For ticket sales, we need some of the speakers confirmed.
+  * Phebe: That's the difference between early bird tickets and regular tickets.
+  * Phebe: CFP goes out start of May. We can use the same system for tickets.
+  * Phebe: Not sure if we're still using sessionize or pretalx. Pretalx doesn't have embeddings.
+  * Phebe: Sponsor recruiting, choosing a chair starts, and a venue. We need a chair by March.
+  * Karen: The chair will take most of the work. Have we mapped out what we need so we can share something with
+           prospects?
+  * Mahmoud: Like a JD.
+  * Phebe: It's basically an 8 month volunteer commitment.
+  * Mahmoud: A month might be on the long side for deliberation.
+  * Phebe: Yes, but it's good if we can have acceptances earlier.
+  * Mahmoud: With the venue, it's a strategic budget question. It should not be up to the chair.
+  * Phebe: Yes, the venue and chair are board decisions.
+  * Mahmoud: What about vendors?
+  * Phebe: Final approval by the board.
+  * Moshe: The chair should do most of the legwork here.
+  * Karen: What if it's a paid position?
+  * Moshe: I don't think we can afford it.
+* Moshe: Choosing a venue seems like we can start now.
+* Phebe: We started looking for places. Maybe we can talk with hotels.
+* Moshe: This might be a problem because we're mostly a local conference.
+* James: How are we going to look for a chair and a venue?
+* Moshe: Can anyone commit to ask quotes etc.?
+* Mahmoud: We have a few people we talked to. I can take care of it.
+* Phebe: Not sure how we can look for a chair.
+* Mahmoud: Is the chair someone already in the community?
+* Mahmoud: Let's collaborate on a list of people in a GDoc.
+* Karen: Yes, we need to make a list of prospects. I'll start the GDoc.
+* Mahmoud: Can Phebe make the JD into a draft?
+* Phebe: Yes, it's leading the meetings, scheduling the meetings, triaging, and being a role model for the community.
+  * Nice to have if it's someone who goes to PyCon US
+  * Day of commitment is to give opening and closing talks, and then post conference to finalize.
+  * Mahmoud: It's a good start. We need to decide how we would evaluate candidate.
+    * Sponsor recruiting at PyCon is good. Maybe we can look into sponsoring a talk.
+* Phebe: Deepset.ai asked about sponsorship.
+* Mahmoud: We should ask Anaconda.
+* Phebe: Yes, and Bloomberg.
+* Mahmoud: Let's do a sponsor leads to GDoc. I'll start it.
```

Comment:
I started it, let's talk about sharing protocol (need to know who to add for perms)